next.config.mjs

```diff
@@ -7,6 +10,11 @@ const nextConfig = {
         hostname: "res.cloudinary.com",
         pathname: "/**",
       },
+      {
+        protocol: "https",
+        hostname: env.NEXT_PUBLIC_GRAPHQL_HOSTNAME || "",
+        pathname: "/**",
```

Comment:
I don't understand the need for this?
We already have the `NEXT_PUBLIC_GRAPHQL_SCHEMA_ENDPOINT_DPL_CMS`